app/helpers/components_helper.rb

```diff
@@ -0,0 +1,13 @@
+module ComponentsHelper
+  def primary_navigation(*args, **kwargs, &block)
+    render PrimaryNavigationComponent.new(*args, **kwargs) do |component|
+      block.call(component) if block.present?
+    end
+  end
+
+  def secondary_navigation(*args, **kwargs, &block)
+    render SecondaryNavigationComponent.new(*args, **kwargs) do |component|
+      block.call(component) if block.present?
+    end
```

Comment:
This might be over my head but I'm not sure what this adds over just calling the component `SecondaryNavigationComponent.new`

Reply:
I've found it to be syntactic sugar really.

![CleanShot 2024-01-08 at 16 30 46](https://github.com/DFE-Digital/itt-mentor-services/assets/4231530/01d3db6b-aafd-48fd-9dc7-f93fff8d5d7d)

I'm not strongly opinionated about either approach though. Happy to support both.